### PR TITLE
docs(react-context-selector): removed the mention of non-existent `useContextSelectors` and fixed the example

### PR DESCRIPTION
**Pull request by Jokero**

### CODE REVIEW

**Comment by layershifter** [comment]
@Jokero can you please run `yarn change` locally and commit & push a generated file? 🙏 Thx

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-context-selector/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # `@fluentui/react-context-selector`
```
Comment (fabricteam) [inline packages/react-components/react-context-selector/README.md:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33146&commitId=3941cbb8d078d56f49cf73bb487227eac06f0a29&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|14| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4933| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by Jokero** [comment]
@microsoft-github-policy-service agree

**Comment by Jokero** [comment]
@layershifter done

**Review by layershifter — APPROVED** [review]

**Pull request merged**